test/Turn-test.js

```diff
@@ -17,24 +17,47 @@ describe('Turn', function() {
   });
 
   it('should store a user guess to the question', function() {
-    const turn = new Turn('my guess');
-    expect(turn.guess).to.equal('my guess');
+    const turn = new Turn('user guess');
+    expect(turn.guess).to.equal('user guess');
   });
 
   it('should store a Card object for the current card in play', function() {
-    const card = new Card()
-    const turn = new Turn('my guess', card)
-    expect(turn.card).to.equal(card)
+    const card = new Card(1, 'What allows you to define a set of related information using key-value pairs?', ['object', 'array', 'function'], 'object');
```

Comment:
I'm thinking it's better to pass in arguments here even though they weren't necessarily needed to pass the test. Seems better practice? I'll have to double check, but I think so.